npm/cypress-schematic/src/builders/cypress/index.ts

```diff
@@ -46,8 +46,8 @@ function runCypress (
     }),
     switchMap((options: CypressBuilderOptions) => {
       return (options.devServerTarget
-        ? startDevServer({ devServerTarget: options.devServerTarget, watch: options.watch, context }).pipe(
-          map((devServerBaseUrl: string) => options.baseUrl || devServerBaseUrl),
+        ? forkJoin(startDevServer({ devServerTarget: options.devServerTarget, watch: options.watch, context })).pipe(
+          map((devServerBaseUrlArray: [string]) => options.baseUrl || devServerBaseUrlArray[0]),
```

Comment:
```suggestion
        ? startDevServer({ devServerTarget: options.devServerTarget, watch: options.watch, context }).pipe(
          map((devServerBaseUrl: string) => options.baseUrl || devServerBaseUrl),
```

Should be able to drop `forkJoin` now that we've refactored the `startDevServer` to return a single observable